src/components/periodic-report/periodic-report.repository.ts

```diff
@@ -454,7 +454,7 @@ export const matchCurrentDue =
       ])
       .raw(`WHERE end.value < date()`)
       .with('node, start')
-      .orderBy('start.value', 'desc')
+      .orderBy('start.value', 'asc')
```

Comment:
I believe this is going to always bring back the earliest/oldest quarter in the range, not the most recent. I just tried testing in my local, but I could be testing it incorrectly.  It always brings the Published report back as the current report due. 
![Screenshot 2024-09-24 at 11 50 04 AM](https://github.com/user-attachments/assets/2e0d46f8-c4e0-43c5-859c-dc5023ad16c4)

Reply:
Bleh 🤦

Reply:
Thank for testing. Can you try this? Now we use the most recently ending report and of the reports with the same end date, the one that starts first.